pyscriptjs/tests/test_altair.py

```diff
@@ -0,0 +1,86 @@
+"""Test: altair"""
+
+import math
+import re
+import time
+
+try:
+    # When running `make test`
+    from .common import MAX_TEST_TIME, TEST_TIME_INCREMENT, _url_join
+except ImportError:
+    # When running `python test_<example name>.py`
+    from common import (
+        MAX_TEST_TIME,
+        TEST_TIME_INCREMENT,
+        _url_join,
+    )
+
+from playwright.sync_api import sync_playwright
+
+BASE_URL = "http://127.0.0.1:8080"
+EXPECTED_TITLE = "Altair"
+HTML_FILE = "altair.html"
+
+
+def test():
+    """Wrapper for all tests for this example so as to use one playwright instance"""
+
+    url = _url_join(BASE_URL, HTML_FILE)
+
+    # Set up playwright package
+    with sync_playwright() as p:
+        browser = p.chromium.launch()
+        page = browser.new_page()
+        page.goto(url)
+
+        # Step 1: Test page title to ensure HTML is loaded properly
+
+        print(f"{HTML_FILE}: Asserting <title></title> == {EXPECTED_TITLE}")
+        assert page.title() == EXPECTED_TITLE  # nosec
+
+        # Step 2: Test that pyodide is loaded via the console
+
+        loading_messages = [
```

Comment:
Checking for these messages may easily break in the near future due to the likelihood that we change stuff but I think it's good for now. It's a good starting point

Reply:
yes, this is a complexity, we need to talk about a better option. playright cannot access the devtools console info. (at least we dont know how to)

one option i want to pitch is having pyodide emit startup info into a hidden dev. so that tooling can get the info